.circleci/define_docker_build_parameters

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
```

Comment:
What do you think of sourcing `.circleci/define_build_environment_variables` here or putting this in to `.circleci/define_build_environment_variables` because `$CLEANED_BRANCH_NAME` comes from `.circleci/define_build_environment_variables` but there doesn't seem to be any relation between the scripts